app/frontend/src/features/messages/groupchats/GroupChatView.test.tsx

```diff
@@ -125,6 +126,79 @@ describe("GroupChatView", () => {
         expect(messageElement.getByText("You created the chat")).toBeVisible();
       }
     }
+    expect(getGroupChatMock).toHaveBeenCalledTimes(1);
+    // 1 is groupChatId here
+    expect(getGroupChatMock).toHaveBeenCalledWith(1);
+    expect(getGroupChatMessagesMock).toHaveBeenCalledTimes(1);
+    expect(getGroupChatMessagesMock).toHaveBeenCalledWith(1, undefined);
+  });
+
+  describe("when there are new messages due to come in", () => {
+    beforeEach(() => {
+      jest.useFakeTimers();
```

Comment:
Lol figuring this out was harder than making the change itself. But glad we have done it so we have an example and also so we don't have to wait for 3 seconds for this test to finish!